cli/lsp/config.rs

```diff
@@ -266,8 +266,15 @@ fn default_to_true() -> bool {
   true
 }
 
+fn empty_string_none<'de, D: serde::Deserializer<'de>>(
+  d: D,
+) -> Result<Option<String>, D::Error> {
+  let o: Option<String> = Option::deserialize(d)?;
+  Ok(o.filter(|s| !s.is_empty()))
```

Comment:
I wonder if we should do `!s.trim().is_empty()`?

Reply:
eh, seems a bit weird